Sources/EduIDExpansion/Flows/Onboarding/OnBoardingMainNavigationDelegate.swift

```diff
@@ -0,0 +1,6 @@
+import UIKit
+
+protocol OnBoardingMainNavigationDelegate: AnyObject {
```

Comment:
See https://github.com/Tiqr/tiqr-app-core-ios/pull/38#discussion_r1100025159

Reply:
I did this